### PR TITLE
Update docs with client side KeepAliveInterval option

### PR DESCRIPTION
**Pull request by mikaelm12**
Fixes: https://github.com/aspnet/AspNetCore.Docs/issues/10783

Review url: https://review.docs.microsoft.com/en-us/aspnet/core/signalr/configuration?view=aspnetcore-2.1&branch=pr-en-us-13669&tabs=dotnet

We were missing the `KeepAliveInterval` option on the client side.

### CODE REVIEW

File: aspnetcore/signalr/configuration.md [hunk 1]
```diff
@@ -327,6 +327,7 @@ Additional options for configuring timeout and keep-alive behavior are available
...
 | `HandshakeTimeout` | 15 seconds | Timeout for initial server handshake. If the server doesn't send a handshake response in this interval, the client cancels the handshake and triggers the `Closed` event (`onclose` in JavaScript). This is an advanced setting that should only be modified if handshake timeout errors are occurring due to severe network latency. For more detail on the Handshake process, see the [SignalR Hub Protocol Specification](https://github.com/aspnet/SignalR/blob/master/specs/HubProtocol.md). |
+| `KeepAliveInterval` | 15 seconds | Determines the interval at which the client sends ping messages. Sending any message from the client resets the timer to the start of the interval. If the client hasn't sent a message in the ClientTimeoutInterval set on the server, the server considers the client disconnected. |
```
Comment (BrennanConroy) [inline aspnetcore/signalr/configuration.md]: These probably need a "moniker range"

Comment (guardrex) [reply aspnetcore/signalr/configuration.md]: If so, moniker version the whole table. We can't do individual rows.

Comment (mikaelm12) [reply aspnetcore/signalr/configuration.md]: Okay, I'll make the old version of the table 2.1 and with the changes will be 2.2 and beyond

Comment (guardrex) [reply aspnetcore/signalr/configuration.md]: Cool ... put the >=2.2 version first, then the <2.2 after it. We version the blocks most recent to oldest.

Comment (guardrex) [reply aspnetcore/signalr/configuration.md]:
```
::: moniker range=">= aspnetcore-2.2"

xxxxxxxxxxxxxx

::: moniker-end

::: moniker range="< aspnetcore-2.2"

xxxxxxxxxxxxxx

::: moniker-end
```

**Pull request merged**